test/es6/ES6Symbol.js

```diff
@@ -972,6 +972,24 @@ var tests = [
             assert.throws(function () { o[Symbol.iterator](); }, TypeError, "Calling non-existent built-in symbol property with description fails", "Object doesn't support property or method 'Symbol(Symbol.iterator)'");
         }
     },
+    {
+        name: "Symbol equality logic",
+        body() {
+            assert.isTrue(Symbol.toPrimitive != 1);
+            assert.isTrue(Symbol.toPrimitive != NaN);
+            
+            var valueOfCalled = false;
+            var a = Symbol('f');
+            var b = {
+                valueOf : function() {
+                    valueOfCalled = true;
+                    return a;
+                }
+            };
+            assert.isTrue(a == b);

```

Comment:
The code here says that a symbol object is equal to a symbol if the symbol wrapped by the symbol object is the same symbol. Does that make sense? Sorry, symbol is in there a bunch.

I think Symbol.prototype[@@toPrimitive] indicates this. If we do toPrimitive of Object(a) and a is a symbol, the result should be a. Yes?